RELEASE_NOTES.md

```diff
@@ -5,8 +5,13 @@ Release Notes {#nf_release_notes}
 
 This file contains a high-level description of this package's evolution.
 Entries are in reverse chronological order (most recent first).
+## 4.5.4 - TBD
 
-## 4.5.3 - TBD
+### Requirements
+
+* netCDF-C: 4.7.4+
+
+## 4.5.3 - April 24, 2020
```

Comment:
Where does this date come from?

Reply:
It was the original target date, but we have been overtaken by circumstances and will be updated when the 4.5.3 release is actually minted.